ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -77,20 +78,23 @@ def test_huber_loss(
         min_dim_size=3,
     ),
     reduction=st.sampled_from(["none", "sum", "batchmean", "mean"]),
-    test_with_out=st.just(False),
+    log_target=st.booleans(),
 )
 def test_kl_div(
     dtype_and_input,
     dtype_and_target,
     reduction,
+    log_target,
     test_flags,
     backend_fw,
     fn_name,
     on_device,
 ):
     input_dtype, input = dtype_and_input
+    input[0] = np.log(input[0])
     target_dtype, target = dtype_and_target
-
+    if log_target:
+        target[0] = np.log(target[0])
```

Comment:
if `log_target` is true then the user is expected to input a log of their target? 🤔

Reply:
Yes that's correct. This is again from [`torch`](https://pytorch.org/docs/stable/generated/torch.nn.functional.kl_div.html) documentation  `A flag indicating whether target is passed in the log space. It is recommended to pass certain distributions (like softmax) in the log space to avoid numerical issues caused by explicit log. Default: False`